apps/dfda/app/auth/signin/page.logged-out.md

```
@@ -0,0 +1,35 @@
+- Open Graph image: https://dfda.earth/assets/dfda/dfda-og-1200x630.png
+- Twitter title: dFDA
+- Twitter description: A decentralized framework for drug assessment for ranking treatments by real-world effectiveness and outcome labels showing the positive and negative effects of every food and drug in the world.
+
+## Visible Page Copy
+
+- [DFDA](/)
+- 🛠️ DEV LOGIN (Test User)
+- Continue with Google
+- Continue with Email
+- EVIDENCE-BASED TREATMENT INFORMATION
+#### TREATMENTS
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)